lib/modules/manager/helmfile/artifacts.ts

```diff
@@ -44,14 +49,15 @@ export async function updateArtifacts({
   try {
     await writeLocalFile(packageFileName, newPackageFileContent);
 
+    const helmfileVersion = parseLock(existingLockFileContent).version;
```

Comment:
inline this, so the lockfile is only parsed when there are no manual constraints. will save time (perf)